pages/app/student-profile/[studentprofile].js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
+
+import { Avatar } from '@mui/material';
+
+import styles from '../../../utils/styles/Profiles.module.css';
+import defaultProfilePic from '../../../utils/constants';
+
+export default function StudentProfile(props) {
+  const [userType, setUserType] = useState('mentor');
+  const [currentStudentRecommend, setCurrentStudentRecommend] = useState(10);
+
+  // increment with axios put request
+  // re render page with new count
+  // add user type: teacher or student
+  const handleUpvoteStudent = () => {
+    setCurrentStudentRecommend(currentStudentRecommend++);
+  }
+
+  const handleDownvoteStudent = () => {
+    setCurrentStudentRecommend(currentStudentRecommend--);
+  }
+
+  return (
+    <div className="pageData">
+      <div className={styles.profileContainer}>
+          <div className={styles.profilePic}>
+            <Avatar
+              className="my-profile-view-avatar"
+              alt="Student Profile Picture"
+              src={defaultProfilePic}
+              sx={{ width: 300, height: 300 }}
```

Comment:
Same as other profile pages. Lmk your thoughts on default profile pic dims in the constants folder